company/values.md

```diff
@@ -1,23 +1,24 @@
 # Sourcegraph values
 
-These values are some of the beliefs and principles that help us achieve our [goals](goals/index.md) and [vision](strategy.md#vision).
+These values are the beliefs and principles that help us achieve our [goals](goals/index.md) and [vision](strategy.md#vision).
 
-This list isn't intended to cover everything we care about; instead, it lists the values that we frequently find useful and refer to. We'll keep this list up to date with the frequently used beliefs and principles (adding, editing, and removing entries as needed). Our hope is that this makes this list more accurate and useful than if it were a list of stale, vague, aspirational, or obvious values.
+- [**Be customer driven**](#be-customer-driven): You proactively work on the right things by continuously orienting your goals around delivering value to your customer.
+- [**Work as a team**](#work-as-a-team): You work collaboratively with your peers, cross-functional teammates, and leadership to create shared success, trust, and belonging.
+- [**High agency**](#high-agency): You have the power and the responsibility to improve Sourcegraph as a company and as a product. You deliver regardless of the circumstances.
+- [**High quality**](#high-quality): Every person on our team is individually responsible for knowing what high-quality work looks like and producing high-quality work.
+- [**Open and transparent**](#open-and-transparent): You proactively communicate in an open and transparent way.
+- [**Continuously grow**](#continuously-grow): You strive to continuously grow and learn by genuinely soliciting feedback early and often, and humbly reflecting on your past mistakes.
+- [**Be human**](#be-human): You are kind, empathetic, authentic, humble.
```

Comment:
Can we move this to the top of the list? 🙂

Reply:
Addressed in https://github.com/sourcegraph/about/pull/1760